### ISSUE TITLE
Fail to render on server

### ISSUE THREAD

**Issue by vuhrmeister** [TeamWertarbyte/mdi-material-ui#17]
I do render my page on the server and get the following error:

```
Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.
```

This is how I import it:

```javascript
import Facebook from 'mdi-material-ui/Facebook'
```

It is however working on the client!

**Comment by leMaik** [comment]
I have no idea what causes this issue for you. Are you sure it's because of the import?

**Comment by vuhrmeister** [comment]
Well, not absolutely sure. But the same implementation is working on the client. Normal Material UI icons are also working fine.

**Comment by vuhrmeister** [comment]
It seems Node can't properly handle the transpiled module.

This is what a console.log shows on the server. First line is the imported component from this library and second from material-ui.

```
{}
{ [Function: ShouldUpdate] displayName: 'pure(Add)', muiName: 'SvgIcon' }
```

Why not just have ES6 modules? I think most of the time code for the client gets transpiled anyway by a bundler.

**Comment by leMaik** [comment]
Many people don't transpile their dependencies. E.g. create-react-app doesn't. I don't know why Node can't handle the transpiled module for you, I've successfully used this library for SSR previously?

**Comment by vuhrmeister** [comment]
Well, 1.4.2
I'm also using Meteor which is using node v8.11.3

**Comment by leMaik** [comment]
Still have no idea, unfortunately. :confused: Might be a Meteor thing?

**Comment by leMaik** [comment]
@vuhrmeister Would your problem be fixed if I publish this as ES6 modules? :thinking:

**Comment by vuhrmeister** [comment]
Might be Meteor, can't say yet. But I'm not using Webpack but the native build system. It doesn't compile code inside of node_modules, so it should import it as is. I rather think it's node itself.

I can't say for sure, but I believe a normal ES6 module should work properly fine since.

I'm just trying to figure out the difference between the Material UI icon module since this is also transpiled.

**Comment by leMaik** [comment]
> But I'm not using Webpack but the native build system.

I'm afraid I can't help you with your build configuration then.

> I'm just trying to figure out the difference between the Material UI icon module since this is also transpiled.

Let me know about any differences you find. I'll update our icons accordingly to make them work for you and other Meteor users.

**Comment by leMaik** [comment]
@vuhrmeister So… i compared the packages

* We use fragments (for... reasons, they are unneccessary and I'll fix that soon). Doesn't matter for this problem
* The transpiled files look pretty much the same (@material-ui/icons uses `@babel/runtime`, while we inline the `_interopRequireDefault` function, but that's also unrelated)
* Material-UI exports all icons as ES6 modules in an `/es` directory. This way, they can be imported as ES6 modules using `import Facebook from '@material-ui/icons/es/Facebook`. Are you doing that? I could add this to our icons, too.

**Comment by vuhrmeister** [comment]
Now you were faster … 
So I just played around a bit and I also found the differences. After few changes I restarted the server so it would import my changed file. But wait – I reverted my changes and it still worked …

Short story: obviously Meteor detects the newly installed npm module while building for the client, but not for server use. To get that fixed it seems I have to restart the Meteor server.

So I must apologize for taking your time but also thank you for your help.

Don't know if importing ES6 modules will be a plus now. Honestly I don't do it for Material UI. But I will try it out. If I see any benefit I will come back to you.
